### ISSUE TITLE
Pipeline Install on cluster

### ISSUE THREAD

**Issue by gitcruz** [blobtoolkit/blobtoolkit#131]
Hi,

I just noticed that installations instructions here are no longer working, because git clone does not find the repositories.
https://blobtoolkit.genomehubs.org/install/

btw, is the snakemake pipeline compatible  with blobtoolkit v3.3.0? which python version do you recommend for the pip install?

Thanks
Fernando

**Comment by maysonlin** [comment]
Hi, I also tried to install locally on my Mac M1 and it is not working anymore by following the instruction. If I followed the flow on the manual (https://blobtoolkit.genomehubs.org/install/), ~/blobtoolkit/blobtool2/blobtool is not there, so I cannot use blobtool create or any blobtool command. I tried to use pip install blobtool, but this error shows up "No matching distribution found for snakemake==7.16.1 (from blobtoolkit)". Any idea how to fix it?

**Comment by sujaikumar** [comment]
Hi all. I apologise - I've been meaning to update the installation instructions at https://blobtoolkit.genomehubs.org/install/

The newer version actually installs much more easily :-) . Here are the steps that I have tried on both an Intel Mac, and on an Ubuntu Linux machine

In both cases, I'd recommend starting with conda because that keeps the environment separate from everything else on your machine, and gets the right version of python, etc:

If you don't already have conda, you'll probably find it VERY useful for lots of other bioinformatics software too! See https://bioconda.github.io/ (and follow the instructions for Miniconda, you don't need Anaconda which is bigger/takes longer)

Then, once you have conda, create a new environment for btk
```
conda create -n btk pip python=3.8
conda activate btk
pip install blobtoolkit
```

That's it :-)

Type `blobtools` to see all the subcommands.

**Comment by margaretc-ho** [comment]
Hi there, I followed the instructions to install using conda (as env named blobtoolkit) and it looks like its installed when I activate the conda environement and run
```
python -m pip install blobtoolkit
Requirement already satisfied: blobtoolkit in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (4.0.7)
Requirement already satisfied: pysam==0.19.1 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (0.19.1)
Requirement already satisfied: geckodriver-autoinstaller==0.1.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (0.1.0)
Requirement already satisfied: chromedriver-binary-auto==0.2.3 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (0.2.3)
Requirement already satisfied: pyvirtualdisplay==3.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (3.0)
Requirement already satisfied: pyyaml in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (6.0)
Requirement already satisfied: tqdm==4.64.1 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (4.64.1)
Requirement already satisfied: blobtoolkit-core>=0.1.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (0.1.2)
Requirement already satisfied: docopt>=0.6.2 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (0.6.2)
Requirement already satisfied: fastjsonschema==2.15.3 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (2.15.3)
Requirement already satisfied: selenium==4.7.2 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (4.7.2)
Requirement already satisfied: tolkein>=0.5.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (0.5.0)
Requirement already satisfied: ujson>=5.7.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (5.7.0)
Requirement already satisfied: psutil==5.9.4 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from blobtoolkit) (5.9.4)
Requirement already satisfied: urllib3[socks]~=1.26 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from selenium==4.7.2->blobtoolkit) (1.26.14)
Requirement already satisfied: trio-websocket~=0.9 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from selenium==4.7.2->blobtoolkit) (0.9.2)
Requirement already satisfied: certifi>=2021.10.8 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from selenium==4.7.2->blobtoolkit) (2022.12.7)
Requirement already satisfied: trio~=0.17 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from selenium==4.7.2->blobtoolkit) (0.22.0)
Requirement already satisfied: requests>=2.24.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from tolkein>=0.5.0->blobtoolkit) (2.28.2)
Requirement already satisfied: xmltodict>=0.12.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from tolkein>=0.5.0->blobtoolkit) (0.13.0)
Requirement already satisfied: charset-normalizer<4,>=2 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from requests>=2.24.0->tolkein>=0.5.0->blobtoolkit) (3.0.1)
Requirement already satisfied: idna<4,>=2.5 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from requests>=2.24.0->tolkein>=0.5.0->blobtoolkit) (3.4)
Requirement already satisfied: attrs>=19.2.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (22.2.0)
Requirement already satisfied: sniffio in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.3.0)
Requirement already satisfied: sortedcontainers in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (2.4.0)
Requirement already satisfied: exceptiongroup>=1.0.0rc9 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.1.0)
Requirement already satisfied: async-generator>=1.9 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.10)
Requirement already satisfied: outcome in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.2.0)
Requirement already satisfied: wsproto>=0.14 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from trio-websocket~=0.9->selenium==4.7.2->blobtoolkit) (1.2.0)
Requirement already satisfied: PySocks!=1.5.7,<2.0,>=1.5.6 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from urllib3[socks]~=1.26->selenium==4.7.2->blobtoolkit) (1.7.1)
Requirement already satisfied: h11<1,>=0.9.0 in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from wsproto>=0.14->trio-websocket~=0.9->selenium==4.7.2->blobtoolkit) (0.14.0)
Requirement already satisfied: typing-extensions in /hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/lib/python3.7/site-packages (from h11<1,>=0.9.0->wsproto>=0.14->trio-websocket~=0.9->selenium==4.7.2->blobtoolkit) (4.5.0)
```
But when I try to run blobtools it gives me the following error
$ blobtools
Traceback (most recent call last):
  File "/hpcdata/bcbb/homc/conda_envs/envs/blobtoolkit/bin/blobtools", line 5, in <module>
    from blobtools import cli
ModuleNotFoundError: No module named 'blobtools'

Any ideas as to why it can't find the blobtools module in Python?

**Comment by rjchallis** [comment]
Version 4 should be run on python 3.8-3.10 (and should work on 3.11). I haven't updated the requirements to formally drop the 3.7 support so will make sure that gets done. 

I was just trying to confirm this and had some issues with pip not pulling the py38 wheel on a Mac (I don't know if that's an issue on linux as well). The containerised versions are running python 3.9 so I'd suggest trying a fresh install with:
```
conda env remove -n blobtoolkit
conda create -n blobtoolkit pip python=3.9
conda activate blobtoolkit
pip install blobtoolkit
```

Hopefully that will run without any problems

**Comment by margaretc-ho** [comment]
Thanks, it works now! 👍

**Comment by chipotlau** [comment]
Hello,

I am running into the same module error: 

I ran 

`conda create -n blobtoolkit pip python=3.9
conda activate blobtoolkit
pip install blobtoolkit`

and here is the pip output:

`pip install blobtoolkit
Requirement already satisfied: blobtoolkit in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (4.1.5)
Requirement already satisfied: tqdm==4.64.1 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (4.64.1)
Requirement already satisfied: chromedriver-binary-auto==0.2.3 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (0.2.3)
Requirement already satisfied: tolkein>=0.5.0 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (0.5.0)
Requirement already satisfied: fastjsonschema==2.15.3 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (2.15.3)
Requirement already satisfied: docopt>=0.6.2 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (0.6.2)
Requirement already satisfied: geckodriver-autoinstaller==0.1.0 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (0.1.0)
Requirement already satisfied: psutil==5.9.4 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (5.9.4)
Requirement already satisfied: selenium==4.7.2 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (4.7.2)
Requirement already satisfied: blobtk>=0.2.3 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (0.3.5)
Requirement already satisfied: pyvirtualdisplay==3.0 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (3.0)
Requirement already satisfied: ujson>=5.7.0 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (5.7.0)
Requirement already satisfied: pyyaml in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from blobtoolkit) (6.0)
Requirement already satisfied: certifi>=2021.10.8 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from selenium==4.7.2->blobtoolkit) (2023.5.7)
Requirement already satisfied: trio-websocket~=0.9 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from selenium==4.7.2->blobtoolkit) (0.10.2)
Requirement already satisfied: trio~=0.17 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from selenium==4.7.2->blobtoolkit) (0.22.0)
Requirement already satisfied: urllib3[socks]~=1.26 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from selenium==4.7.2->blobtoolkit) (1.26.16)
Requirement already satisfied: requests>=2.24.0 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from tolkein>=0.5.0->blobtoolkit) (2.31.0)
Requirement already satisfied: xmltodict>=0.12.0 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from tolkein>=0.5.0->blobtoolkit) (0.13.0)
Requirement already satisfied: charset-normalizer<4,>=2 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from requests>=2.24.0->tolkein>=0.5.0->blobtoolkit) (3.1.0)
Requirement already satisfied: idna<4,>=2.5 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from requests>=2.24.0->tolkein>=0.5.0->blobtoolkit) (3.4)
Requirement already satisfied: outcome in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.2.0)
Requirement already satisfied: async-generator>=1.9 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.10)
Requirement already satisfied: sortedcontainers in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (2.4.0)
Requirement already satisfied: exceptiongroup>=1.0.0rc9 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.1.1)
Requirement already satisfied: sniffio in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.3.0)
Requirement already satisfied: attrs>=19.2.0 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (23.1.0)
Requirement already satisfied: wsproto>=0.14 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from trio-websocket~=0.9->selenium==4.7.2->blobtoolkit) (1.2.0)
Requirement already satisfied: PySocks!=1.5.7,<2.0,>=1.5.6 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from urllib3[socks]~=1.26->selenium==4.7.2->blobtoolkit) (1.7.1)
Requirement already satisfied: h11<1,>=0.9.0 in ./anaconda3/envs/blobtoolkit/lib/python3.9/site-packages (from wsproto>=0.14->trio-websocket~=0.9->selenium==4.7.2->blobtoolkit) (0.14.0)`

and here is the error:

`$ blobtools
Traceback (most recent call last):
  File "/Users/emilylau/anaconda3/envs/blobtoolkit/bin/blobtools", line 5, in <module>
    from blobtools import cli
ModuleNotFoundError: No module named 'blobtools'`

I am using MacOS 13.3.1 M2 chip — any suggestions would be greatly appreciated! Thanks again for developing this tool.

**Comment by sujaikumar** [comment]
Hi @chipotlau - could you try `pip install —no-cache-dir blobtoolkit`

**Comment by chipotlau** [comment]
Thanks for your fast reply! I really appreciate your help. I tried running `pip install —no-cache-dir blobtoolkit` in my blobtoolkit conda environment and got the same module error.

However, I ran `pip3 install —no-cache-dir blobtoolkit` using python3 installed from homebrew instead of conda, which allowed me to successfully install and run blobtools:

`pip3 install --no-cache-dir blobtoolkit
Collecting blobtoolkit
  Downloading blobtoolkit-4.1.5-py311-none-macosx_11_0_arm64.whl (111 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 111.7/111.7 kB 5.3 MB/s eta 0:00:00
Collecting blobtk>=0.2.3
  Downloading blobtk-0.3.5-cp311-cp311-macosx_10_9_x86_64.macosx_11_0_arm64.macosx_10_9_universal2.whl (2.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.4/2.4 MB 5.7 MB/s eta 0:00:00
Collecting chromedriver-binary-auto==0.2.3
  Downloading chromedriver-binary-auto-0.2.3.tar.gz (5.1 kB)
  Preparing metadata (setup.py) ... done
Collecting docopt>=0.6.2
  Downloading docopt-0.6.2.tar.gz (25 kB)
  Preparing metadata (setup.py) ... done
Collecting fastjsonschema==2.15.3
  Downloading fastjsonschema-2.15.3-py3-none-any.whl (22 kB)
Collecting geckodriver-autoinstaller==0.1.0
  Downloading geckodriver_autoinstaller-0.1.0-py3-none-any.whl (5.6 kB)
Collecting psutil==5.9.4
  Downloading psutil-5.9.4-cp38-abi3-macosx_11_0_arm64.whl (244 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 244.2/244.2 kB 22.0 MB/s eta 0:00:00
Collecting pyvirtualdisplay==3.0
  Downloading PyVirtualDisplay-3.0-py3-none-any.whl (15 kB)
Collecting pyyaml
  Downloading PyYAML-6.0-cp311-cp311-macosx_11_0_arm64.whl (167 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 167.5/167.5 kB 23.3 MB/s eta 0:00:00
Collecting selenium==4.7.2
  Downloading selenium-4.7.2-py3-none-any.whl (6.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.3/6.3 MB 19.4 MB/s eta 0:00:00
Collecting tolkein>=0.5.0
  Downloading tolkein-0.5.0-py3-none-any.whl (13 kB)
Collecting tqdm==4.64.1
  Downloading tqdm-4.64.1-py2.py3-none-any.whl (78 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.5/78.5 kB 30.1 MB/s eta 0:00:00
Collecting ujson>=5.7.0
  Downloading ujson-5.7.0-cp311-cp311-macosx_11_0_arm64.whl (53 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 53.7/53.7 kB 166.8 MB/s eta 0:00:00
Collecting urllib3[socks]~=1.26
  Downloading urllib3-1.26.16-py2.py3-none-any.whl (143 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 143.1/143.1 kB 32.9 MB/s eta 0:00:00
Collecting trio~=0.17
  Downloading trio-0.22.0-py3-none-any.whl (384 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 384.9/384.9 kB 39.3 MB/s eta 0:00:00
Collecting trio-websocket~=0.9
  Downloading trio_websocket-0.10.2-py3-none-any.whl (17 kB)
Collecting certifi>=2021.10.8
  Downloading certifi-2023.5.7-py3-none-any.whl (156 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 157.0/157.0 kB 34.9 MB/s eta 0:00:00
Collecting requests>=2.24.0
  Downloading requests-2.31.0-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.6/62.6 kB 30.6 MB/s eta 0:00:00
Collecting xmltodict>=0.12.0
  Downloading xmltodict-0.13.0-py2.py3-none-any.whl (10.0 kB)
Collecting charset-normalizer<4,>=2
  Downloading charset_normalizer-3.1.0-cp311-cp311-macosx_11_0_arm64.whl (121 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 121.7/121.7 kB 46.5 MB/s eta 0:00:00
Collecting idna<4,>=2.5
  Downloading idna-3.4-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 224.4 MB/s eta 0:00:00
Collecting urllib3<3,>=1.21.1
  Downloading urllib3-2.0.2-py3-none-any.whl (123 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 123.2/123.2 kB 38.2 MB/s eta 0:00:00
Collecting attrs>=19.2.0
  Downloading attrs-23.1.0-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.2/61.2 kB 28.1 MB/s eta 0:00:00
Collecting sortedcontainers
  Downloading sortedcontainers-2.4.0-py2.py3-none-any.whl (29 kB)
Collecting async-generator>=1.9
  Downloading async_generator-1.10-py3-none-any.whl (18 kB)
Collecting outcome
  Downloading outcome-1.2.0-py2.py3-none-any.whl (9.7 kB)
Collecting sniffio
  Downloading sniffio-1.3.0-py3-none-any.whl (10 kB)
Collecting exceptiongroup
  Downloading exceptiongroup-1.1.1-py3-none-any.whl (14 kB)
Collecting wsproto>=0.14
  Downloading wsproto-1.2.0-py3-none-any.whl (24 kB)
Collecting PySocks!=1.5.7,<2.0,>=1.5.6
  Downloading PySocks-1.7.1-py3-none-any.whl (16 kB)
Collecting h11<1,>=0.9.0
  Downloading h11-0.14.0-py3-none-any.whl (58 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.3/58.3 kB 270.4 MB/s eta 0:00:00
Building wheels for collected packages: chromedriver-binary-auto, docopt
  Building wheel for chromedriver-binary-auto (setup.py) ... done
  Created wheel for chromedriver-binary-auto: filename=chromedriver_binary_auto-0.2.3-py3-none-any.whl size=8409852 sha256=10dfa8199203a641382e7c1ba82a7b762d1bd04688df190f8ad8aa051207e65b
  Stored in directory: /private/var/folders/zx/26cmg_4n0893lw1czyjmxd640000gn/T/pip-ephem-wheel-cache-e7mypply/wheels/c7/9e/91/ca17b5dc092950f4578e4b675b6dae3e4ec3cc3c5cf80d52c0
  Building wheel for docopt (setup.py) ... done
  Created wheel for docopt: filename=docopt-0.6.2-py2.py3-none-any.whl size=13707 sha256=eeaa74d6639f734af1c64d875adce14e03e0020f417f2f60640ea4f696373867
  Stored in directory: /private/var/folders/zx/26cmg_4n0893lw1czyjmxd640000gn/T/pip-ephem-wheel-cache-e7mypply/wheels/1a/b0/8c/4b75c4116c31f83c8f9f047231251e13cc74481cca4a78a9ce
Successfully built chromedriver-binary-auto docopt
Installing collected packages: sortedcontainers, pyvirtualdisplay, fastjsonschema, docopt, chromedriver-binary-auto, xmltodict, urllib3, ujson, tqdm, sniffio, pyyaml, PySocks, psutil, idna, h11, geckodriver-autoinstaller, exceptiongroup, charset-normalizer, certifi, blobtk, attrs, async-generator, wsproto, requests, outcome, trio, tolkein, trio-websocket, selenium, blobtoolkit
Successfully installed PySocks-1.7.1 async-generator-1.10 attrs-23.1.0 blobtk-0.3.5 blobtoolkit-4.1.5 certifi-2023.5.7 charset-normalizer-3.1.0 chromedriver-binary-auto-0.2.3 docopt-0.6.2 exceptiongroup-1.1.1 fastjsonschema-2.15.3 geckodriver-autoinstaller-0.1.0 h11-0.14.0 idna-3.4 outcome-1.2.0 psutil-5.9.4 pyvirtualdisplay-3.0 pyyaml-6.0 requests-2.31.0 selenium-4.7.2 sniffio-1.3.0 sortedcontainers-2.4.0 tolkein-0.5.0 tqdm-4.64.1 trio-0.22.0 trio-websocket-0.10.2 ujson-5.7.0 urllib3-1.26.16 wsproto-1.2.0 xmltodict-0.13.0

[notice] A new release of pip is available: 23.0.1 -> 23.1.2
[notice] To update, run: python3.11 -m pip install --upgrade pip`

After installing, here is the output of the blob tools command:

`blobtools

BlobTools2 - assembly exploration, QC and filtering.

usage: blobtools [<command>] [<args>...] [-h|--help] [--version]

commands:
    add             add data to a BlobDir
    create          create a new BlobDir
    filter          filter a BlobDir
    host            host interactive view of all BlobDirs in a directory
    replace         call blobtools add with --replace flag
    remove          remove one or more fields from a BlobDir
    validate        validate a BlobDir
    view            generate plots using BlobToolKit Viewer
    -h, --help      show this
    -v, --version   show version number
See 'blobtools <command> --help' for more information on a specific command.

examples:
    # 1. Create a new BlobDir from a FASTA file:
    blobtools create --fasta examples/assembly.fasta BlobDir

    # 2. Create a new BlobDir from a BlobDB:
    blobtools create --blobdb examples/blobDB.json BlobDir

    # 3. Add Coverage data from a BAM file:
    blobtools add --cov examples/assembly.reads.bam BlobDir

    # 4. Assign taxonomy from BLAST hits:
    blobtools add add --hits examples/blast.out --taxdump ../taxdump BlobDir

    # 5. Add BUSCO results:
    blobtools add --busco examples/busco.tsv BlobDir

    # 6. Host an interactive viewer:
    blobtools host BlobDir

    # 7. Filter a BlobDir:
    blobtools filter --param length--Min=5000 --output BlobDir_len_gt_5000 BlobDir`

I am not sure why the blobtoolkit conda environment did not work, perhaps there was a conflict between python3 in homebrew and conda? But, I believe it is working now.. Thanks again!
